awscli/customizations/scalarparse.py

```diff
@@ -36,13 +39,29 @@ def identity(x):
     return x
 
 
+def iso_format(value):
+    return parse_timestamp(value).isoformat()
+
+
+def choose_default_parsers(timestamp_format):
```

Comment:
I wonder if it makes sense to have this choose all of the parsers. It seems a little strange that it sets both the blob and timestamp options, but really you can only pass in a ``timestamp_format`` option. Wonder if it make sense to have a function:
```python
def add_timestamp_parser(session):
     # Sets the timestamp parser to the session based on the config
```
That way you can encompass all of the timestampy related functionality under one function and makes it easier to read.